web/src/routes/__tests__/PoisPage.spec.tsx

```diff
@@ -51,30 +42,30 @@ describe('PoisPage', () => {
         pathname={pathname}
         languageCode={language.code}
         cityCode={city.code}
-      />
-    </ThemeProvider>
-  )
+      />,
+      { wrapWithTheme: true }
+    )
 
   it('should render a list with all pois', () => {
-    mockUseLoadFromEndpointWithData(pois)
-    const { getByText } = renderWithRouter(renderPois)
+    mocked(useFeatureLocations).mockImplementation(() => ({
+      data: prepareFeatureLocations(pois, null),
+      loading: false,
+      error: null,
+      refresh: jest.fn()
+    }))
+    const { getByText } = renderPois()
     expect(getByText(poi0.location.name)).toBeTruthy()
     expect(getByText(poi1.location.name)).toBeTruthy()
   })
 
   it('should render an error', () => {
-    mockUseLoadFromEndpointWithError('Something went wrong')
-    const { getByText } = renderWithRouter(renderPois)
+    mocked(useFeatureLocations).mockImplementation(() => ({
+      data: null,
+      loading: false,
+      error: new Error('Something went wrong'),
+      refresh: jest.fn()
+    }))
+    const { getByText } = renderPois()
     expect(getByText('error:unknownError')).toBeTruthy()
   })
-
-  it('should render poi details page when list item was clicked', () => {
```

Comment:
I reevaluated this test, to see whether we could somehow make it work. Therby I found, that flyTo doesn't work for me in web in edge. Furthermore, I now know that this test fails, as the MapRef is always null. But when trying to mock/spy it you'll break the searchparams. I think somehow this is a really tricky logic and we maybe need to refactor this on once more at a later point. For now I'd suggest to maybe revert the additional work that was put into this PR. Although I know you put quite some work into this.